extensions/amp-lightbox-viewer/0.1/test/test-amp-lightbox-viewer.js

```diff
@@ -50,6 +50,23 @@ describes.realWin('amp-lightbox-viewer', {
     doc.body.appendChild(viewer);
```

Comment:
The main change here is to put all the stubs in the `getAmpLightboxViewer` function so it doesn't have to be repeated below. Adds a stub for `mutatePromise` since we took out `mutate` and put in `mutatePromise` for the `buildCarousel_` function.